### PR TITLE
Russian language update

### PR DESCRIPTION
**Pull request by rkislov**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: resources/lang/ru.json [hunk 1]
```diff
@@ -1,69 +1,256 @@
...
-    "Network": "Сеть",
-    "Device Groups Maps": "Карта сгруппированных устройств",
-    "Geographical": "Географичесая",
-    "Plugins": "Дополнения",
-    "Plugin Admin": "Управление дополнениями",
-    "Tools": "Инструменты",
-    "Eventlog": "Хронология",
-    "Inventory": "Инвенторизация",
-    "MIB definitions": "Настройка MIB",
+    "The :attribute must a valid IP address\/network or hostname.": ":attribute lдолжен быть правильным IP адресом\/подсетью или именем устройства.",
```
Comment (zombah) [inline resources/lang/ru.json]: maybe s/lдолжен/должен/g

File: resources/lang/ru.json [hunk 2]
```diff
@@ -1,69 +1,256 @@
...
+    "Never polled": "Неразу не опрашивался",
```
Comment (zombah) [inline resources/lang/ru.json]: I think spelling error here ,```Ни разу``` or  ```Никогда``` maybe

File: resources/lang/ru.json [hunk 3]
```diff
@@ -1,69 +1,256 @@
...
+    "Edit User": "Изменить пользователя",
+    "Unlock": "Разблокировать",
+    "User exceeded failures": "Пользователь допустил много ошибок",
+    "Disable TwoFactor": "Отключить двуфакторную аутентификацию",
+    "No TwoFactor key generated for this user, Nothing to do.": "Не сгенерирован ключ для двуфакторной аутентификации.",
+    "Save": "Сохранить",
+    "Cancel": "Отмена",
+    "Unlocked Two Factor.": "Разблокирован двуфакторная аутентификация.",
+    "Failed to unlock Two Factor": "Не удалось разблокировать двухфакторную аутентификацию",
+    "Removed Two Factor.": "Убрана второй фактор.",
```
Comment (zombah) [inline resources/lang/ru.json]: Sounds strange

File: resources/lang/ru.json [hunk 4]
```diff
@@ -1,69 +1,256 @@
...
+    "Description": "Описание",
+    "Level": "Уровень",
+    "Normal": "Нормально",
+    "Global Read": "Глобальное Чтение",
+    "Admin": "Admin",
+    "Demo": "Demo",
+    "Dashboard": "Панель",
+    "Password": "Пароль",
+    "Current Password": "Действующий пароль",
+    "New Password": "новый пароль",
```
Comment (zombah) [inline resources/lang/ru.json]: English start with upper case letter here

File: resources/lang/ru.json [hunk 5]
```diff
@@ -1,69 +1,256 @@
...
+    "Logon": "Вход",
+    "Device": "Устройство",
+    "Platform": "Платформа",
+    "Uptime": "Аптайм",
+    "Location": "Расположение",
+    "Status": "Статус",
+    "Remember Me": "Запомнить меня",
+    "Login": "Логин",
+    "Please enter auth token": "Пожалуйста введите токен авторизации",
+    "Submit": "выполнить",
```
Comment (zombah) [inline resources/lang/ru.json]: English starts with upper case letter here

File: resources/lang/ru.json [hunk 6]
```diff
@@ -1,69 +1,256 @@
...
+    "Logout": "Выйти",
+    "Locations": "Располодения",
```
Comment (zombah) [inline resources/lang/ru.json]: Spelling error

File: resources/lang/ru.json [hunk 7]
```diff
@@ -1,69 +1,256 @@
...
+    "Custom title for widget": "Изменяемый заголовок для виджета",
+    "Display type": "Отображаемый тип",
+    "boxes": "боксы",
+    "compact": "компактный",
+    "Uniform Tiles": "Равномерная плитка",
+    "Tile size": "Размер заголовка",
+    "Disabled\/ignored": "Выключенно\/игнорированно",
+    "Show": "Показать",
+    "Hide": "Скрыть",
+    "Mode select": "Выюор режима",
```
Comment (zombah) [inline resources/lang/ru.json]: Spelling error

File: resources/lang/ru.json [hunk 8]
```diff
@@ -1,69 +1,256 @@
...
+    "Two Months": "Два месяца",
+    "Three Months": "Три месяца",
+    "One Year": "Один Год",
+    "Two Years": "Два Года",
+    "Select a device": "Выберите устройство",
+    "Port": "Порт",
+    "Select a port": "Выберите порт",
+    "Application": "Приложение",
+    "Select an application": "Выберите приложение",
+    "Munin plugin": "Munin plugin",
```
Comment (zombah) [inline resources/lang/ru.json]: Above ```plugin``` in ```Munin plugin``` phrase was translated

File: resources/lang/ru.json [hunk 9]
```diff
@@ -1,69 +1,256 @@
...
+    "Select a Munin plugin": "Выберите Munin plugin",
+    "Bill": "Счет",
+    "Select a bill": "Выберите счйт",
+    "Custom Aggregator(s)": "Пользовательский аггрегатор(ций)",
+    "Select or add one or more": "Выберите илидобавьте один или более",
```
Comment (zombah) [inline resources/lang/ru.json]: Missing space

File: resources/lang/ru.json [hunk 10]
```diff
@@ -1,69 +1,256 @@
...
-    "Alert Transports": "Транспорт извещений",
-    "My Settings": "Мои настройки",
-    "Settings": "настройки",
-    "Global Settings": "Глобальные настройки",
-    "Validate Config": "Проверка конфигурации",
-    "Manage Users": "Управление пользователями",
-    "Auth History": "История авторизации",
+    "Errored": "Ошибок",
+    "Services": "Сервисы",
+    "No devices found within interval.": "В жтот интервал не попало ни одного устройсктва.",
```
Comment (zombah) [inline resources/lang/ru.json]: Spelling error

File: resources/lang/ru.json [hunk 11]
```diff
@@ -1,69 +1,256 @@
...
+    "Summary": "Суммарно",
+    "Interface": "Интерфейс",
+    "Total traffic": "Общий трафик",
+    "Check your log for more details.": "Порверьте лог для большей информации..",
```
Comment (zombah) [inline resources/lang/ru.json]: Spelling error

**Comment by rkislov** [comment]
@zombah thank you for spell checking. I'll update my pull request.

**Pull request merged**